depthai_nodes/ml/parsers/classification.py

```diff
@@ -20,32 +21,57 @@ class ClassificationParser(dai.node.ThreadedHostNode):
         Parser sends the processed network results to this output in a form of DepthAI message. It is a linking point from which the processed network results are retrieved.
     classes : List[str]
         List of class names to be used for linking with their respective scores. Expected to be in the same order as Neural Network's output. If not provided, the message will only return sorted scores.
-    is_softmax : bool = True
-        If False, the scores are converted to probabilities using softmax function.
     n_classes : int = len(classes)
         Number of provided classes. This variable is set automatically based on provided classes.
+    is_softmax : bool = True
+        If False, the scores are converted to probabilities using softmax function.
 
     Output Message/s
     ----------------
     **Type** : Classifications(dai.Buffer):
          An object with attributes `classes` and `scores`. `classes` is a list of classes, sorted in descending order of scores. `scores` is a list of corresponding scores.
     """
 
-    def __init__(self, classes: List[str] = None, is_softmax: bool = True):
+    def __init__(self):
         """Initializes the ClassificationParser node.
 
-        @param classes: List of class names to be used for linking with their respective
-            scores.
-        @param is_softmax: If False, the scores are converted to probabilities using
-            softmax function.
+        Attributes
```

Comment:
It makes sense to document that those only get populated after `build()` is called.